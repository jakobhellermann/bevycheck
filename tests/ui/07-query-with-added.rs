#![allow(unused)]
use bevy::prelude::*;

struct Foo;

#[bevycheck::system]
fn system(_: Query<&Foo, With<Added<Foo>>>) {}

fn main() {
    App::build().add_system(system.system()).run();
}
