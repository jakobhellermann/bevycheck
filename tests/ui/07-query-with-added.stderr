error: invalid query filter
 --> $DIR/07-query-with-added.rs:7:26
  |
7 | fn system(_: Query<&Foo, With<Added<Foo>>>) {}
  |                          ^^^^^^^^^^^^^^^^
  |
  = note: `With` should be used like `With<Component>
